<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>George C. Linderman's Homepage</title>

<link rel="stylesheet" href="./styles.css">
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111570395-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111570395-1');
</script>
</head>

<body>
<div id="info">
<img src="./gcl2.jpg" width="240" alt="Picture of Linderman" title="George Linderman">
<h1>George C. Linderman</h1> 
<ul>
  <li> General Surgery Resident </li>
  <li> <a href="https://www.massgeneral.org/surgery">Department of Surgery</a> </li>
   <li> <a href="https://www.massgeneral.org/">Massachusetts General Hospital</a> </li>
  <li><b> Email</b>: <a href="mailto:glinderman[at]mgh.harvard.edu"> glinderman[at]mgh.harvard.edu</a></li>
  <li><b> Twitter</b>: <a href="https://twitter.com/GCLinderman">@GCLinderman</a></li>
  </ul>
</div>
<div id="contents">

<h2>About me:</h2> 
<p> I completed my MD PhD training at Yale University, where I pursued a PhD in Applied
Mathematics. Broadly speaking, my work has been in developing theory and methods for high dimensional data analysis. </p>

<p> At the present time, I am focusing on clinical training as a surgical resident at the Massachusetts General Hospital.</p>

</div>
<div>
<h2>Software</h2>
</div>
I maintain the following software packages:
<ul>

	<li><a href="https://github.com/KlugerLab/FIt-SNE">FIt-SNE</a>: Fast Fourier Transform-accelerated Interpolation-based t-SNE</li>
	<li><a href="https://github.com/KlugerLab/ALRA">ALRA</a>: Adaptively-thresholded Low Rank Approximation for imputation of scRNA-seq data</li>
	<li><a href="https://github.com/KlugerLab/t-SNE-Heatmaps">t-SNE Heatmaps</a>: 1D t-SNE heatmaps to visualize expression patterns of hundreds of genes simultaneously in scRNA-seq</li>
	<li><a href="https://github.com/KlugerLab/oocPCA">oocPCA</a>: Out-of-Core Randomized Principal Component Analysis</li>
	<li><a href="https://github.com/KlugerLab/pcafast">pcafast</a>: Randomized Principal Component Analysis</li>
</ul>

<h2>Teaching</h2>
</div>
<ul>

	<li>High Dimensional Probability (Directed Reading Program Mentor, Fall 2018, Spring 2019, Fall 2019, Fall 2020)</li>
	<li>MATH222: Linear Algebra with Applications (Teaching Fellow, Fall 2017)</li>
	

<div>
<h2>Notes</h2>
</div>
<!-- Below are notes I wrote about topics I found enlightening: -->
<ul>
	<li><a href="https://gclinderman.github.io/blog/numerics/low-rank/t-sne/2018/01/11/low-rank-kernels.html">Low Rank Approximation of Kernels</a>: Intuition for the interpolation methods used to accelerate t-SNE</li>
	<li><a href="https://gclinderman.github.io/blog/probability/2018/01/07/concentration-inequalities.html">Concentration Inequalities</a>: An introduction to concentration inequalities, with numerical demonstrations </li>
	<li><a href="https://gclinderman.github.io/blog/statistics/probability/matrix/completion/2018/07/08/matrix-completion.html">Low-Rank Matrix Completion</a>: An introduction to low-rank matrix completion</li>
</ul>
<div>
<div>
<h2>Selected publications</h2> 
</div>
Papers where I was lead/co-lead author are listed below; see my <a href="https://scholar.google.com/citations?user=u0AG8GAAAAAJ&hl=en">Google Scholar</a> page for complete list of publications.
<br /> 
<br /> 

<b>Methods and Theory Papers</b>

<ul class='papers'>
	<li> 
            <a href="https://www.nature.com/articles/s41587-020-00809-z">Initialization is critical for preserving global data structure in both t-SNE and UMAP</a> (<i>Nature Biotechnology</i>, 2021)
	</li>
	<li> 
            <a href="https://link.springer.com/protocol/10.1007%2F978-1-0716-1307-8_18">Dimensionality Reduction of Single-Cell RNA-Seq Data</a> (<i>Methods in Molecular Biology</i>, 2021)
	</li>
	<li> 
            <a href="https://www.nature.com/articles/s41592-018-0308-4">Fast interpolation-based t-SNE for improved visualization of single-cell RNA-seq data</a> (<i>Nature Methods</i>, 2019)
	</li>
	<li> 
		<a href="https://www.biorxiv.org/content/early/2018/08/22/397588">Zero-preserving imputation of scRNA-seq data using low rank approximation</a> (<i>bioRxiv</i>, 2018)
	</li>
	<li> 
            <a href = "https://www.ams.org/journals/mcom/2020-89-324/S0025-5718-2020-03515-7/">Numerical Integration on Graphs: Where to Sample and How to Weigh</a> (<i>Mathematics of Computation</i>, 2018, <a href="https://arxiv.org/abs/1803.06989">arXiv</a>)
	</li>
        <li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7480951/">Randomized Near Neighbor Graphs, Giant Components, and Applications in Data Science</a> (<i>Journal of Applied Probability</i>, 2018, <a href="https://arxiv.org/abs/1711.04712">arXiv</a>)</li>
        <li> 
            <a href="https://epubs.siam.org/doi/abs/10.1137/18M1216134">Clustering with t-SNE, Provably</a> (<i>SIAM Journal on Mathematics of Data Science</i>, 2017)
        </li>
<li><a href="https://dl.acm.org/citation.cfm?id=3004053">Algorithm 971: An implementation of a randomized algorithm for principal component analysis</a> (<i>ACM Transactions of Mathematical Software</i>, 2017)

</li>
<li><a href="https://academic.oup.com/nar/article/40/W1/W152/1079514">MAGNET: MicroArray Gene expression and Evaluation Toolkit</a> (<i>Nucleic Acids Research</i>, 2012) </li>
<li><a href="https://academic.oup.com/bioinformatics/article/27/8/1174/227848">BiC: a Web Server for Calculating Bimodality of Coexpression Between Gene and Protein Networks.</a> (<i>Bioinformatics</i>, 2011) </li>
</ul>


<br />
<b> Applications Papers</b>
<ul class="papers">
<li> 
   <a href="https://www.surgjournal.com/article/S0039-6060(21)00681-4/fulltext">Improved outcomes using laparoscopy for emergency colectomy after mitigating bias by negative control exposure analysis</a> (<i>Surgery</i>, 2021) 
</li>
<li> 
   <a href="https://journals.lww.com/jtrauma/abstract/9000/increased_mortality_with_reboa_only_mitigated_by.97438.aspx">Increased mortality with REBOA only mitigated by strong unmeasured confounding: an expanded analysis using the National Trauma Data Bank</a> (<i>Journal of Trauma and Acute Care Surgery</i>, 2021) 
</li>
<li> 
   <a href="https://journals.lww.com/jbjsjournal/Fulltext/2019/10160/The_Proximal_Humeral_Ossification_System_Improves.10.aspx">The Proximal Humeral Ossification System Improves Assessment of Maturity in Patients with Scoliosis</a> (<i>Journal of Joint and Bone Surgery</i>, 2019) 
</li>
<li> 
    <a href="https://jamanetwork.com/journals/jamanetworkopen/fullarticle/2696872">Association of Body Mass Index With Blood Pressure Among 1.7 Million Chinese Adults</a> (<i>JAMA Network Open</i>, 2018)
</li>
</ul>

<!---
<ol>
	<li> 
		Kobak, D., <u>Linderman, G. C.</u>, Steinerberger, S., Kluger, Y., Berens, P. (2019). Heavy-tailed kernels reveal a finer cluster structure in t-SNE visualisations.
	<a href="https://arxiv.org/abs/1902.05804">arXiv</a>.
	</li>
	<li> 
            <u>Linderman, G. C.</u>, Rachh, M., Hoskins, J.G., Steinerberger, S., Kluger, Y. (2019). Fast interpolation-based t-SNE for improved visualization of single-cell RNA-seq data. <i> Nature Methods.</i> 
	<a href="https://www.nature.com/articles/s41592-018-0308-4">link</a>, <a href="https://arxiv.org/abs/1712.09005">arXiv</a>.
	</li>
	<li> 
		<u>Linderman, G. C.</u>, Zhao, J., Kluger, Y. (2018). Zero-preserving imputation of scRNA-seq data using low rank approximation.
	<a href="https://www.biorxiv.org/content/early/2018/08/22/397588">bioRxiv</a>.
	</li>
	<li> 
            <u>Linderman, G. C.</u>, Lu, J., Lu, Y., Sun, X., Xu, W., Nasir, K., Schulz, W., Jiang, L., Krumholz, H.M.  Association of Body Mass Index With Blood Pressure Among 1.7 Million Chinese Adults. (2018). <i>JAMA Network Open.</i> 2018. <a href="https://jamanetwork.com/journals/jamanetworkopen/fullarticle/2696872">link</a>.
	</li>
	<li> 
		<u>Linderman, G. C.</u>, Steinerberger, S. (2018). Numerical Integration on Graphs: Where to Sample and How to Weigh.
	<a href="https://arxiv.org/abs/1803.06989">arXiv</a>.
	</li>
	<li>Lu, J., Lu, Y., Wang, X., Li, X., <u>Linderman, G.C.</u>, Wu, C., Cheng, X., Mu, L., Zhang, H., Liu, J., Su, M., Zhao, H., Spatz, E.S., Spertus, J.A., Masoudi, F.A., Krumholz, H.M., Jiang, L. Underdetected and Undertreated Hypertension in China: Data from 1.7 Million Adults in the China PEACE Million Persons Project. (2017). <i>Lancet</i>.  <a href="https://www.sciencedirect.com/science/article/pii/S0140673617324789">link</a>.</li>
	<li><u>Linderman, G.C.</u>, Mishne, G., Kluger, Y., Steinerberger, S. (2017). Randomized Near Neighbor Graphs, Giant Components, and Applications in Data Science. <a href="https://arxiv.org/abs/1711.04712">arXiv</a>.</li>
<li> 
    <u>Linderman, G. C.</u>, Steinerberger, S. (2017). Clustering with t-SNE, Provably. <i>SIAM Journal on Mathematics of Data Science</i>. <a href="https://epubs.siam.org/doi/abs/10.1137/18M1216134">link</a>.
</li>
<li>Li, H., <u>Linderman, G.C.</u> , Szlam, A., Stanton, K.P., Kluger, Y., Tygert, M. (2017). “Algorithm 971: An implementation of a randomized algorithm for principal component analysis.” <i>ACM Transactions of Mathematical Software.</i>  43, 3, Article 28.
<a href="https://dl.acm.org/citation.cfm?id=3004053">link</a>.
</li>
<li>Wang Z, Jenkins M, <u>Linderman G.C.</u>, Bezerra H, Fujino Y, Costa M, Wilson D, Rollins, A. “3-D Stent Detection in Intravascular OCT Using a Bayesian Network and Graph Search.” <i>IEEE Transactions on Medical Imaging.</i> 34 (7) 1549-1561. <a href="http://ieeexplore.ieee.org/abstract/document/7047878/">link</a>.</li>
<li><u>Linderman, G.C.</u>, Chance, M.R., Bebek, G.: “MAGNET: MicroArray Gene expression and Evaluation Toolkit.” <i>Nucleic Acids Research </i>40 (W1), W152-W156. <a href="https://academic.oup.com/nar/article/40/W1/W152/1079514">link</a>.</li>
<li><u>Linderman, G.C.</u>, Patel, V.N., Chance, M.R., Bebek, G.: “BiC: a Web Server for Calculating Bimodality of Coexpression Between Gene and Protein Networks.” <i>Bioinformatics</i> 27 (8), 1174-1175. <a href="https://academic.oup.com/bioinformatics/article/27/8/1174/227848">link</a>. </li>


</ol>

-->
 
</body></html>
