<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>George C. Linderman's Homepage</title>

<link rel="stylesheet" href="./styles.css">
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111570395-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111570395-1');
</script>
</head>

<body>
<div id="info">
<img src="./gcl2.jpg" width="240" alt="Picture of Linderman" title="George Linderman">
<h1>George C. Linderman</h1> 
<ul>
  <li> MD PhD Student </li>
  <li> <a href="https://applied.math.yale.edu/">Program in Applied Mathematics</a> </li>
   <li> <a href="http://www.yale.edu/">Yale University</a> </li>
   <li> <b>Office</b>: AKW 112</li>
  <li><b> Email</b>: <a href="mailto:george.linderman[at]yale.edu"> george.linderman[at]yale.edu </a></li>
  </ul>
</div>
<div id="contents">

<h2>About me:</h2> 
I am an MD PhD student at Yale University, and I have completed my PhD in Applied
Mathematics. Broadly, I am interested in developing mathematical and computational
solutions to biomedical problems.

</div>
<div>
<h2>Software</h2>
</div>
I maintain the following software packages:
<ul>

	<li><a href="https://github.com/KlugerLab/FIt-SNE">FIt-SNE</a>: Fast Fourier Transform-accelerated Interpolation-based t-SNE</li>
	<li><a href="https://github.com/KlugerLab/ALRA">ALRA</a>: Adaptively-thresholded Low Rank Approximation for imputation of scRNA-seq data</li>
	<li><a href="https://github.com/KlugerLab/t-SNE-Heatmaps">t-SNE Heatmaps</a>: 1D t-SNE heatmaps to visualize expression patterns of hundreds of genes simultaneously in scRNA-seq</li>
	<li><a href="https://github.com/KlugerLab/oocPCA">oocPCA</a>: Out-of-Core Randomized Principal Component Analysis</li>
	<li><a href="https://github.com/KlugerLab/pcafast">pcafast</a>: Randomized Principal Component Analysis</li>
</ul>

<h2>Teaching</h2>
</div>
<ul>

	<li>High Dimensional Probability (Directed Reading Program Mentor, Fall 2018, Spring 2019, Fall 2019, Fall 2020)</li>
	<li>MATH222: Linear Algebra with Applications (Teaching Fellow, Fall 2017)</li>
	

<div>
<h2>Notes</h2>
</div>
<!-- Below are notes I wrote about topics I found enlightening: -->
<ul>
	<li><a href="http://gauss.math.yale.edu/~gcl22/blog/numerics/low-rank/t-sne/2018/01/11/low-rank-kernels.html">Low Rank Approximation of Kernels</a>: Intuition for the interpolation methods used to accelerate t-SNE</li>
	<li><a href="http://gauss.math.yale.edu/~gcl22/blog/probability/2018/01/07/concentration-inequalities.html">Concentration Inequalities</a>: An introduction to concentration inequalities, with numerical demonstrations </li>
	<li><a href="https://gauss.math.yale.edu/~gcl22/blog/statistics/probability/matrix/completion/2018/07/08/matrix-completion.html">Low-Rank Matrix Completion</a>: An introduction to low-rank matrix completion</li>
</ul>
<div>
<div>
<h2>Publications & preprints</h2> 
</div>

<a href="https://scholar.google.com/citations?user=u0AG8GAAAAAJ&hl=en">Publication list on Google Scholar</a>

<br />
<br />
<b>Methods and Theory Papers</b>
<ol>
	<li> 
            Kobak, D., <u>Linderman, G.C.</u> (2020). UMAP does not preserve global structure any better than t-SNE when using the same initialization. <a href="https://www.biorxiv.org/content/10.1101/2019.12.19.877522v1.abstract">bioRxiv</a>.
	</li>
	<li> 
            Kobak, D., <u>Linderman, G.C.</u>, Steinerberger, S., Kluger, Y., Berens, P. (2019). Heavy-tailed kernels reveal a finer cluster structure in t-SNE visualisations. <i>ECML.</i> <a href ="https://link.springer.com/chapter/10.1007/978-3-030-46150-8_8">link</a>,
	<a href="https://arxiv.org/abs/1902.05804">arXiv</a>.
	</li>
	<li> 
            <u>Linderman, G.C.</u>, Rachh, M., Hoskins, J.G., Steinerberger, S., Kluger, Y. (2019). Fast interpolation-based t-SNE for improved visualization of single-cell RNA-seq data. <i> Nature Methods.</i> 
	<a href="https://www.nature.com/articles/s41592-018-0308-4">link</a>, <a href="https://arxiv.org/abs/1712.09005">arXiv</a>.
	</li>
	<li> 
		<u>Linderman, G.C.</u>, Zhao, J., Kluger, Y. (2018). Zero-preserving imputation of scRNA-seq data using low rank approximation.
	<a href="https://www.biorxiv.org/content/early/2018/08/22/397588">bioRxiv</a>.
	</li>
	<li> 
            <u>Linderman, G.C.</u>, Steinerberger, S. (2018). Numerical Integration on Graphs: Where to Sample and How to Weigh. <i>Mathematics of Computation</i>. <a href = "https://www.ams.org/journals/mcom/2020-89-324/S0025-5718-2020-03515-7/">link</a>,
	<a href="https://arxiv.org/abs/1803.06989">arXiv</a>.
	</li>
	<li><u>Linderman, G.C.</u>, Mishne, G., Kluger, Y., Steinerberger, S. (2017). Randomized Near Neighbor Graphs, Giant Components, and Applications in Data Science. <a href="https://arxiv.org/abs/1711.04712">arXiv</a>.</li>
<li> 
    <u>Linderman, G.C.</u>, Steinerberger, S. (2017). Clustering with t-SNE, Provably. <i>SIAM Journal on Mathematics of Data Science</i>. <a href="https://epubs.siam.org/doi/abs/10.1137/18M1216134">link</a>.
</li>
<li>Li, H., <u>Linderman, G.C.</u> , Szlam, A., Stanton, K.P., Kluger, Y., Tygert, M. (2017). “Algorithm 971: An implementation of a randomized algorithm for principal component analysis.” <i>ACM Transactions of Mathematical Software.</i>  43, 3, Article 28.
<a href="https://dl.acm.org/citation.cfm?id=3004053">link</a>.
</li>
<li>Wang Z, Jenkins M, <u>Linderman G.C.</u>, Bezerra H, Fujino Y, Costa M, Wilson D, Rollins, A. (2015). “3-D Stent Detection in Intravascular OCT Using a Bayesian Network and Graph Search.” <i>IEEE Transactions on Medical Imaging.</i> <a href="http://ieeexplore.ieee.org/abstract/document/7047878/">link</a>.</li>
<li><u>Linderman, G.C.</u>, Chance, M.R., Bebek, G. (2012).  “MAGNET: MicroArray Gene expression and Evaluation Toolkit.” <i>Nucleic Acids Research</i>. <a href="https://academic.oup.com/nar/article/40/W1/W152/1079514">link</a>.</li>
<li><u>Linderman, G.C.</u>, Patel, V.N., Chance, M.R., Bebek, G. (2011). “BiC: a Web Server for Calculating Bimodality of Coexpression Between Gene and Protein Networks.” <i>Bioinformatics</i>. <a href="https://academic.oup.com/bioinformatics/article/27/8/1174/227848">link</a>. </li>
</ol>


<br />
<b> Applications Papers</b>
<ol>
<li> 
    Li, D.T., <u>Linderman, G.C.</u>,   Cui, J.J., DeVries, S.,  Nicholson, A.D., Li, Eric., Petit, L.,  Kahan, J.B., Talty, R., Kluger, Y., Cooperman, D.R., Smith, B.G. (2019). The Proximal Humeral Ossification System Improves Assessment of Maturity in Patients with Scoliosis.  <i>Journal of Joint and Bone Surgery</i>. <a href="https://journals.lww.com/jbjsjournal/Fulltext/2019/10160/The_Proximal_Humeral_Ossification_System_Improves.10.aspx">link</a>.
</li>
<li> 
    Gupta, K., Levinsohn, J., <u>Linderman, G.C.</u>, Chen, D., Sun, T.Y., Dong, D., Taketo, M.M., Bosenberg, M., Kluger, Y., Choate, K. and Myung, P., (2019). Single-cell analysis reveals a hair follicle dermal niche molecular differentiation trajectory that begins prior to morphogenesis. <i>Developmental Cell.</i> <a href="https://www.sciencedirect.com/science/article/pii/S1534580718309882">link</a>.
</li>
<li> 
    <u>Linderman, G.C.</u>, Lu, J., Lu, Y., Sun, X., Xu, W., Nasir, K., Schulz, W., Jiang, L., Krumholz, H.M. (2018).  Association of Body Mass Index With Blood Pressure Among 1.7 Million Chinese Adults. <i>JAMA Network Open.</i> <a href="https://jamanetwork.com/journals/jamanetworkopen/fullarticle/2696872">link</a>.
</li>
<li>Lu, J., Lu, Y., Wang, X., Li, X., <u>Linderman, G.C.</u>, Wu, C., Cheng, X., Mu, L., Zhang, H., Liu, J., Su, M., Zhao, H., Spatz, E.S., Spertus, J.A., Masoudi, F.A., Krumholz, H.M., Jiang, L. (2017). Underdetected and Undertreated Hypertension in China: Data from 1.7 Million Adults in the China PEACE Million Persons Project. <i>Lancet</i>.  <a href="https://www.sciencedirect.com/science/article/pii/S0140673617324789">link</a>.</li>
</ol>

<!---
<ol>
	<li> 
		Kobak, D., <u>Linderman, G. C.</u>, Steinerberger, S., Kluger, Y., Berens, P. (2019). Heavy-tailed kernels reveal a finer cluster structure in t-SNE visualisations.
	<a href="https://arxiv.org/abs/1902.05804">arXiv</a>.
	</li>
	<li> 
            <u>Linderman, G. C.</u>, Rachh, M., Hoskins, J.G., Steinerberger, S., Kluger, Y. (2019). Fast interpolation-based t-SNE for improved visualization of single-cell RNA-seq data. <i> Nature Methods.</i> 
	<a href="https://www.nature.com/articles/s41592-018-0308-4">link</a>, <a href="https://arxiv.org/abs/1712.09005">arXiv</a>.
	</li>
	<li> 
		<u>Linderman, G. C.</u>, Zhao, J., Kluger, Y. (2018). Zero-preserving imputation of scRNA-seq data using low rank approximation.
	<a href="https://www.biorxiv.org/content/early/2018/08/22/397588">bioRxiv</a>.
	</li>
	<li> 
            <u>Linderman, G. C.</u>, Lu, J., Lu, Y., Sun, X., Xu, W., Nasir, K., Schulz, W., Jiang, L., Krumholz, H.M.  Association of Body Mass Index With Blood Pressure Among 1.7 Million Chinese Adults. (2018). <i>JAMA Network Open.</i> 2018. <a href="https://jamanetwork.com/journals/jamanetworkopen/fullarticle/2696872">link</a>.
	</li>
	<li> 
		<u>Linderman, G. C.</u>, Steinerberger, S. (2018). Numerical Integration on Graphs: Where to Sample and How to Weigh.
	<a href="https://arxiv.org/abs/1803.06989">arXiv</a>.
	</li>
	<li>Lu, J., Lu, Y., Wang, X., Li, X., <u>Linderman, G.C.</u>, Wu, C., Cheng, X., Mu, L., Zhang, H., Liu, J., Su, M., Zhao, H., Spatz, E.S., Spertus, J.A., Masoudi, F.A., Krumholz, H.M., Jiang, L. Underdetected and Undertreated Hypertension in China: Data from 1.7 Million Adults in the China PEACE Million Persons Project. (2017). <i>Lancet</i>.  <a href="https://www.sciencedirect.com/science/article/pii/S0140673617324789">link</a>.</li>
	<li><u>Linderman, G.C.</u>, Mishne, G., Kluger, Y., Steinerberger, S. (2017). Randomized Near Neighbor Graphs, Giant Components, and Applications in Data Science. <a href="https://arxiv.org/abs/1711.04712">arXiv</a>.</li>
<li> 
    <u>Linderman, G. C.</u>, Steinerberger, S. (2017). Clustering with t-SNE, Provably. <i>SIAM Journal on Mathematics of Data Science</i>. <a href="https://epubs.siam.org/doi/abs/10.1137/18M1216134">link</a>.
</li>
<li>Li, H., <u>Linderman, G.C.</u> , Szlam, A., Stanton, K.P., Kluger, Y., Tygert, M. (2017). “Algorithm 971: An implementation of a randomized algorithm for principal component analysis.” <i>ACM Transactions of Mathematical Software.</i>  43, 3, Article 28.
<a href="https://dl.acm.org/citation.cfm?id=3004053">link</a>.
</li>
<li>Wang Z, Jenkins M, <u>Linderman G.C.</u>, Bezerra H, Fujino Y, Costa M, Wilson D, Rollins, A. “3-D Stent Detection in Intravascular OCT Using a Bayesian Network and Graph Search.” <i>IEEE Transactions on Medical Imaging.</i> 34 (7) 1549-1561. <a href="http://ieeexplore.ieee.org/abstract/document/7047878/">link</a>.</li>
<li><u>Linderman, G.C.</u>, Chance, M.R., Bebek, G.: “MAGNET: MicroArray Gene expression and Evaluation Toolkit.” <i>Nucleic Acids Research </i>40 (W1), W152-W156. <a href="https://academic.oup.com/nar/article/40/W1/W152/1079514">link</a>.</li>
<li><u>Linderman, G.C.</u>, Patel, V.N., Chance, M.R., Bebek, G.: “BiC: a Web Server for Calculating Bimodality of Coexpression Between Gene and Protein Networks.” <i>Bioinformatics</i> 27 (8), 1174-1175. <a href="https://academic.oup.com/bioinformatics/article/27/8/1174/227848">link</a>. </li>


</ol>

-->
 
</body></html>
